end-to-end-applications/kotlin/food-ordering/webui/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# pull official base image
+FROM node:14.17.3-alpine
+
+# set working directory
+WORKDIR /app
+
+# add `/node_modules/.bin` to $PATH
+ENV PATH /node_modules/.bin:$PATH
+
+# install app dependencies
+COPY package.json ./
+COPY package-lock.json ./
```

Comment:
When trying to run the Docker compose file, it failed on this line because I didn't do `npm install` yet and `package-lock.json` didn't exist. So either remove this line and just let it get generated by `npm install` on the next line, or commit `package-lock.json` as well to the repo.